coil-sample-view/src/main/java/coil/sample/ImageListAdapter.kt

```diff
@@ -12,6 +12,9 @@ import androidx.recyclerview.widget.RecyclerView
 import coil.load
 import coil.memory.MemoryCache
 import coil.sample.ImageListAdapter.ViewHolder
+import sample.common.Image
```

Comment:
Yes, this would be done once we set up the baseline for the view as well. No harm in doing now also but I don't want to bloat this PR. Either way, I am fine doing this. WDYT?

Reply:
Good point, let's save it for later.